packages/create-platformatic/package.json

```diff
@@ -13,7 +13,7 @@
     "create-platformatic": "./create-platformatic.mjs"
   },
   "scripts": {
-    "test": "standard | snazzy && NODE_OPTIONS=--loader=esmock c8 --100 tap --no-coverage test/*test.mjs test/*/*test.mjs",
+    "test": "standard | snazzy && NODE_OPTIONS=\"--loader=esmock --no-warnings\" c8 --100 tap --no-coverage test/*test.mjs test/*/*test.mjs",
```

Comment:
Does it work to pass `NODE_OPTIONS` to `tap` directly instead of passing them to `c8` as well? Not a big deal if not.

Reply:
I don't think it would be possible.